### PR TITLE
gocd/rabbit-openqa.py: Always include flavor, name and machine in the check

### PR DESCRIPTION
**Pull request by Vogtinator**
When including images in staging checks we get conflicting names as we have e.g. `microos@uefi-staging` for both `Staging-DVD` and `MicroOS-Image-ContainerHost`.

### CODE REVIEW

**Comment by Vogtinator** [comment]
The required checks in existing staging projects hardcode the simplified check names, so this would break them. Before merging this that needs to be adjusted.

For now I'll create a separate PR which keeps the simplified name in case there is no conflict.

**Comment by coolo** [comment]
Not a fan. "openqa:MicroOS-Image-ContainerHost-microos@uefi-staging" is quite a beast of a check name. But as I don't have to look at the staging dashboard all day, I'll leave that to @DimStar77, @suntorytimed, @gyr and @sysrich and peers I forgot.

**Comment by Vogtinator** [comment]
> Not a fan. "openqa:MicroOS-Image-ContainerHost-microos@uefi-staging" is quite a beast of a check name. But as I don't have to look at the staging dashboard all day, I'll leave that to @DimStar77, @suntorytimed, @gyr and @sysrich and peers I forgot.

FTR, for SLE the longest name would be `Online-Y-Staging-rescue_system_sle11sp4@64bit` - not that bad.

**Comment by coolo** [comment]
plus the openqa: prefix. Simulating in Factory dashbord it would give 
image

I'm just asking questions!

**Review by coolo — APPROVED** [review]

**Comment by Vogtinator** [comment]
@suntorytimed, @gyr

Your input is still missing. If there are no objections, I'd like to merge this next week.

**Pull request merged**
